src/js/debug.js

```diff
@@ -786,6 +786,7 @@ const DEBUG = {
         { text: "Altitude" },
         { text: "RSSI" },
         { text: "Gyro" },
+        { text: "Attitude" },
```

Comment:
```suggestion
```

For backwards compatibility: add attitude around line 816/817 (within semver block):
```
DEBUG.enableFields.splice(DEBUG.enableFields.indexOf("Gyro"), 0, "Attitude");
```

Reply:
The lines sequence are changed after rebase.
Can not understand, what i can do..